scripts/globals/job_utils/monk.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- Monk Job Utilities
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/settings")
+require("scripts/globals/status")
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.monk = xi.job_utils.monk or {}
+
+-----------------------------------
+-- Ability Check Functions
+-----------------------------------
+xi.job_utils.monk.checkHundredFists = function(player, target, ability)
+    ability:setRecast(ability:getRecast() - player:getMod(xi.mod.ONE_HOUR_RECAST))
+    return 0, 0
+end
+
+xi.job_utils.monk.checkInnerStrength = function(player, target, ability)
+    ability:setRecast(ability:getRecast() - player:getMod(xi.mod.ONE_HOUR_RECAST))
+    return 0, 0
+end
+
+-----------------------------------
+-- Ability Use Functions
+-----------------------------------
+xi.job_utils.monk.useBoost = function(player, target, ability)
+    local power = 12.5 + (0.10 * player:getMod(xi.mod.BOOST_EFFECT))
+
+    if player:hasStatusEffect(xi.effect.BOOST) then
+        local effect = player:getStatusEffect(xi.effect.BOOST)
+        effect:setPower(effect:getPower() + power)
+        player:addMod(xi.mod.ATTP, power)
+    else
+        player:addStatusEffect(xi.effect.BOOST, power, 0, 180)
+    end
+end
+
+xi.job_utils.monk.useChakra = function(player, target, ability)
+    local chakraStatusEffects =
```

Comment:
Just for clarity, could this be defined outside of the function in this global?

Reply:
Since the variable is only used for Chakra I’d put it into Chakra’s function but can definitely move it out into the global. Should it go above all of the other ability functions or directly above Chakra’s function?